sms_sponsorship/tests/test_sms_compassion.py

```diff
@@ -19,12 +19,112 @@ class TestSmsCompassion(BaseSponsorshipTest):
 
     def setUp(self):
         super(TestSmsCompassion, self).setUp()
-        self.child = self.create_child('UG1119182')
-        self.child.hold_id.channel = 'sms'
+        self.child = self.env['compassion.child'].search([
+            ('id', '=', 5)
+        ])
+        # self.child.hold_id.channel = 'sms'
+        self.partner = self.env['res.partner'].search([
+            ('id', '=', 15)
```

Comment:
Not a good practice, you should use self.env.ref() instead and find a xml_id in the demo data of base module.